source/Background/Controller.js

```diff
@@ -147,11 +147,12 @@ const secureController = async (callback, controller) => {
   }
 };
 
-init();
-
-// Exposing module methods
-const connectionModule = new ConnectionModule(backgroundController, secureController, keyring);
-connectionModule.exposeMethods();
+let connectionModule;
+init().then(() => {
```

Comment:
Sounds like a good idea +1 I'll make sure to keep this logic when I create the BackgroundScript module

Reply:
we should maybe transform everything into a class instead of having code lying around tbh